src/dawn/Compiler/Options.inc

```diff
@@ -40,6 +40,10 @@ OPT(std::string, Backend, "gridtools", "backend", "",
     "\n - c++-naive   = naive C++ code\n"
     "\n - c++-opt     = optimized C++ code"
     "\n - cuda        = optimized cuda", "<backend>", true, false)
+OPT(std::string, LoadSerialized, "", "load-serialized", "",
+    "Load the serialized data from <file> and only run the passes enabled after deserialization", "<file>", true, false)
+OPT(bool, SerializeIIR, false, "serialize-iir", "",
```

Comment:
for serializing sir, we have -write-sir and here -serialize-iir

I think we have a consistent way of doing. Perhaps 
-serialize=iir
-serialize=sir

Reply:
I also would like to be able to serialize in  Byte format (so adding the option)

Reply:
sounds good-
done

Reply:
sorry, didn't respond to the first comment:

we should keep the wording between the two consistent, I agree. But the two are coming from different places. It makes no sense that the dawn-compiler can serialize the SIR as this is what he gets passed. This should be a frontend functionality whereas the IIR should be serializable from dawn

Reply:
yes, I agree. But can we then make gtclang option and dawn option consistent (and choose which one will be)?